docs/guides/latest/workloads/deployment.md

```diff
@@ -0,0 +1,435 @@
+
+```console
+$ watch -n 2 kubectl get backupsession -n demo 
+Every 1.0s: kubectl get backupsession -n demo     suaas-appscode: Mon Jun 24 10:23:08 2019
+
+NAME                           BACKUPCONFIGURATION   PHASE       AGE
+deployment-backup-1561350065   deployment-backup     Running     30s
+deployment-backup-1561350125   deployment-backup     Succeeded   63s
+```
+
+We can see above that the backup session has succeeded. Now, we will verify that the backed up data has been stored in the local backend.
+
+**Verify Backup:**
+
+Once a backup is complete, Stash will update the respective `Repository` crd to reflect the backup. Check that the repository `local-repo` has been updated by the following command,
+
+```console
+ $ kubectl get repository -n demo local-repo
+NAME         INTEGRITY   SIZE   SNAPSHOT-COUNT   LAST-SUCCESSFUL-BACKUP   AGE
+local-repo   true        0 B    1                27s                      1m4s
+```
+
+## Restore Deployment's Volumes
+
+This section will show you how to restore the backed up data from the backend we have taken in earlier section.
+
+**Deploy Deployment:**
+
+Now, we have to deploy a new Deployment whether the backed up data has been restored.
+
+Below is the YAML of the Deployment that we are going to create,
+
+```console
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: demo-pvc
+  namespace: demo
+spec:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: standard
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: stash-demo
+  name: stash-recovered
+  namespace: demo
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: stash-demo
+  template:
+    metadata:
+      labels:
+        app: stash-demo
+      name: busybox
+    spec:
+      containers:
+      - args:
+        - sleep
+        - "3600"
+        image: busybox
+        imagePullPolicy: IfNotPresent
+        name: busybox
+        volumeMounts:
+        - mountPath: /source/data
+          name: source-data
+      restartPolicy: Always
+      volumes:
+      - name: source-data
+        persistentVolumeClaim:
+          claimName: demo-pvc
+```
+
+Let's create the deployment we have shown above.
+
+```console
+$ kubectl apply -f ./recovered_deployment.yaml 
+persistentvolumeclaim/demo-pvc created
+deployment.apps/stash-recovered created
+```
+
+Now, wait for deployment’s pod to go into the `Running` state.
+
+```console
+$ kubectl get pod -n demo 
+NAME                               READY   STATUS    RESTARTS   AGE
+stash-recovered-676bd87957-7bcfg   1/1     Running   0          31s
+stash-recovered-676bd87957-pv4hg   1/1     Running   0          31s
+stash-recovered-676bd87957-q2hbl   1/1     Running   0          31s
+
+```
+
+**Create RestoreSession:**
+
+We need to create a `RestoreSession` crd to restore the backed up data in the target volume from backend.
```

Comment:
Now, we need to create a `RestoreSession` crd targeting the `stash-recovered` Deployment to restore the inside it.